Python/sysmodule.c

```diff
@@ -921,6 +970,31 @@ sys_exit_impl(PyObject *module, PyObject *status)
 }
 
 
+// This function is used to warn about the future change of sys.abiflags
+// from absent to a meaningful value on all platforms.
+// It can be removed when the change is made.
+static PyObject *
+sys___getattr__(PyObject *module, PyObject *name)
+{
+    PyObject *value = NULL;
+    if (_PySys_GetOptionalAttr(name, &value) < 0) {
+        return NULL;
+    }
+    if (value == NULL) {
+        PyErr_Format(PyExc_AttributeError,
+                     "module 'sys' has no attribute '%U'", name);
+    }
+    return value;
+}
+
+PyDoc_STRVAR(__getattr___doc,
```

Comment:
Avoid declaring variables whose name is prefixed by double underscores; those are reserved by the C compiler, so instead use something like `module__getattr__doc` for instance or `sysmodule___getattr__doc`.